### PR TITLE
Added README

### PR DESCRIPTION
**Pull request by kurtseifried**
This is the README for the OSV-GSD schema information

### CODE REVIEW

File: gsd-schema/osv-gsd/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# OSV GSD Schema
+
+The GSD project primarily relies on the OSV schema for data, but we use an enhanced version of the currently released OSV schema. The reason for this is that the OSV takes time to release new versions of the schema, and in order to prevent blocking the release of data, we use the extended version which is backward compatible with the OSV schema. Using the standard OSV parser will result in warnings (e.g. some ENUM values we use are not present in OSV), but simply ignore the extended fields and attributes you will be able to process the data.
+
+# OSV GSD Extensions
+
+## severity["type": "CVSS_V2"]
+
+Upstream: https://github.com/ossf/osv-schema/pull/87
+Released in OSV schema: 1.4.0
```
Comment (joshbuker) [inline gsd-schema/osv-gsd/README.md:10]: Anything that gets released in upstream should be removed from the list, imo.

File: gsd-schema/osv-gsd/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Released in OSV schema: 1.5.0
+
+We need to explicitly know what type of data this is, as do downstream consumers.
+
+## {"affected": ["ranges": ["events": ["last_affected": "VALUE"]]]}
+
+Upstream: https://github.com/ossf/osv-schema/pull/147
+Released in OSV schema: 1.5.0
+
+We need to explicitly set what the last affected version is for software like the Linux Kernel.
```
Comment (joshbuker) [inline gsd-schema/osv-gsd/README.md:33]: How is this distinct from fixed?

**Review by joshbuker — CHANGES_REQUESTED** [review]
We should only include things that have not been released, imo.

Also, we should add further clarification for `last_affected`, and how it's different/distinct from `fixed`.

File: gsd-schema/osv-gsd/README.md (hunk 1 above)
Comment (kurtseifried) [reply gsd-schema/osv-gsd/README.md:10]: No, we show everything that has been accepted, basically new stuff at the bottom, old stuff at the top.

File: gsd-schema/osv-gsd/README.md (hunk 2 above)
Comment (kurtseifried) [reply gsd-schema/osv-gsd/README.md:33]: last_affected is the last affected version, there may not be a fixed version yet (if ever).

**Review by joshbuker — APPROVED** [review]

**Pull request merged**